### PR TITLE
fix items not being able to be placed on ancient altar

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

This bug apparenttly has existed for 7 months.
On 1.19.4 you are not able to place items on an ancient altar you get the following error
https://pastebin.com/87UpB46w

## Proposed changes

revert https://github.com/Slimefun/Slimefun4/commit/060b59dd0f09886b72d0b4e63ab9f3d7ce5db952

## Related Issues (if applicable)

issue reported on discord
https://canary.discord.com/channels/565557184348422174/565569196218384385/1196756895948619797

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [ ] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.16.* - 1.20.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Comment by WalshyDev** [comment]
Let's version this, I'd rather not bring back flicker for 1.20 and all future versions.

File: src/main/java/io/github/thebusybiscuit/slimefun4/utils/ArmorStandUtils.java [hunk 1]
```diff
@@ -47,13 +49,22 @@ public class ArmorStandUtils {
...
-        return location.getWorld().spawn(location, ArmorStand.class, armorStand -> {
-            armorStand.setVisible(false);
-            armorStand.setSilent(true);
-            armorStand.setMarker(true);
-            armorStand.setGravity(false);
-            armorStand.setBasePlate(false);
-            armorStand.setRemoveWhenFarAway(false);
-        });
+        // 1.19 and below don't have the consumer method so flicker exists on these versions.
+        if (Slimefun.getMinecraftVersion().isBefore(MinecraftVersion.MINECRAFT_1_20)) {
```
Comment (WalshyDev) [inline src/main/java/io/github/thebusybiscuit/slimefun4/utils/ArmorStandUtils.java:53]: I wish we had a isAtMost but we don't haha

**Comment by JustAHuman-xD** [comment]
I'm kinda confused I'll admit, I made this PR working in 1.19, I'm going to check Java docs some more cause I'm currently looking at some that say the method exists for spigot 1.19.1 at least

**Comment by JustAHuman-xD** [comment]
Exists on 1.16.5 too, checked using: https://jd.andross.fr

Simply search "spawn(Location" and you'll find it

File: src/main/java/io/github/thebusybiscuit/slimefun4/utils/ArmorStandUtils.java (hunk 1 above)
Comment (J3fftw1) [reply src/main/java/io/github/thebusybiscuit/slimefun4/utils/ArmorStandUtils.java:53]: We have isatleast

**Review by JustAHuman-xD — APPROVED** [review]
Approving for now till we figure out a way to deal with this, seems to be broken by a signature change

**Comment by WalshyDev** [comment]
Discord convo for ref: https://canary.discord.com/channels/565557184348422174/565569196218384385/1196787007125471273

**Pull request merged**